client/js/lounge.js

```diff
@@ -111,6 +111,47 @@ $(function() {
 		index: 1
 	};
 
+	const colorCodeMap = [
+		[0, "White"],
+		[1, "Black"],
+		[2, "Blue"],
+		[3, "Green"],
+		[4, "Red"],
+		[5, "Brown"],
+		[6, "Magenta"],
+		[7, "Orange"],
+		[8, "Yellow"],
+		[9, "Light Green"],
+		[10, "Cyan"],
+		[11, "Light Cyan"],
+		[12, "Light Blue"],
+		[13, "Pink"],
+		[14, "Grey"],
+		[15, "Light Grey"],
+	];
+
+	const colorStrategy = {
+		id: "colors",
+		match: /^\x03(\d{0,2}|[A-Za-z ]{0,10})$/,
+		search(term, callback) {
+			term = term.toLowerCase();
+			const matchingColorCodes = colorCodeMap
+				.filter(i => i[0].toString().startsWith(term) || i[1].toLowerCase().startsWith(term));
+
+			callback(matchingColorCodes);
+		},
+		template(value) {
+			if (value === colorCodeMap[0]) {
```

Comment:
Duh, of course 😊
But I actually went for white on default background, for several reasons: white on black looks gross in terms of UI, I reuse the text color for the background color dropdown text, the hovering background shows text alright.

Only thing is hovering background is veeeery dim, we really need to change it someday :D